src/Test/Utilities/Portable/Compilation/TestOperationWalker.cs

```diff
@@ -537,5 +515,22 @@ public override void VisitInvalidExpression(IInvalidExpression operation)
         {
             base.VisitInvalidExpression(operation);
         }
+
+        private void VisitArgumentsInParameterOrder(IHasArgumentsExpression operation, ImmutableArray<IParameterSymbol>? parameters)
```

Comment:
This thread (https://github.com/dotnet/roslyn/pull/18494#pullrequestreview-31684803) is still active.